plugins/powerd/fu-plugin-powerd.c

```diff
@@ -65,40 +65,39 @@ fu_plugin_destroy(FuPlugin *plugin)
 		g_object_unref(data->proxy);
 }
 
-static gboolean
-fu_plugin_powerd_refresh_cb(FuPlugin *plugin)
+static void
+fu_plugin_powerd_rescan(GVariant *parameters, FuPlugin *plugin)
```

Comment:
I don't really like having the GVariant first.  Nearly every other use that passes the plugin as an argument, the plugin is the first argument.  Would you mind swapping them?

Reply:
yeah fs
would you also like it to be swapped so plugin is first on the callback function?

Reply:
You cannot do that... as plugin is holding the gpointer user_data expected in the signature of the callback

Reply:
I could swap g_signal_connect() for g_signal_connect_swapped() with the same arguments and it would just put the user_data first (to my understanding of the function)

Reply:
fair enough, I think I rather the g_signal_connect() use to be consistent with other parts of the code (as the upower plugin) but I'll defer to Mario if he thinks otherwise

Reply:
I agree with @djcampello let's stay consistent with other parts of the code.